resourcemanager/resourceids/parse.go

```diff
@@ -238,7 +262,7 @@ func (p Parser) parseSegment(segment Segment, rawValue string, insensitively boo
 				return &*segment.FixedValue, nil
```

Comment:
Not touched here, but I think this should be simplified to: 
```suggestion
				return segment.FixedValue, nil
```